stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -384,6 +384,7 @@ function get_value(sym::Expr, fn)
 end
 get_value(sym::Symbol, fn) = isdefined(fn, sym) ? (getfield(fn, sym), true) : (nothing, false)
 get_value(sym::QuoteNode, fn) = isdefined(fn, sym.value) ? (getfield(fn, sym.value), true) : (nothing, false)
+get_value(sym::GlobalRef, fn) = get_value(sym.name, sym.mod)
```

Comment:
I added this method because `Core.@cmd` macro will be lowered with `GlobalRef`, but this one looks a bit weird since it totally ignores the context `fn`.
Maybe special-casing it inside `isexpr(sym, :macrocall)` block within `get_type` looks better ?

Reply:
This seems like the correct thing to do to me. Of course, we would expect all `GlobalRef`s emitted by the parser to actually exist, but I don't think being more defensive here hurts.

Reply:
okay, then I will leave this as is.